jenkins/test-history/gen_html.py

```diff
@@ -185,6 +230,34 @@ def write_html(outdir, path, html):
         buf.write(html)
 
 
+def transpose(data):
```

Comment:
It's harder to make incremental changes to the data in test-major format -- pruning old builds is particularly frustrating.